Topics/Tech_Stacks/DigitalOcean_Web_App.md

```diff
@@ -0,0 +1,129 @@
+# Deploying a Web Application on DigitalOcean
+
+## Introduction
+This tutorial will guide you step-by-step through the process of deploying a web application using DigitalOcean's hosting platform. DigitalOcean, a cloud service provider, provides a straightforward and user-friendly environment for hosting applications, making it an excellent choice for developers to use.
+
+## Prerequisites
+Before you begin, ensure you have the following:
+
+1. **DigitalOcean Account**: Sign up for a DigitalOcean account if you don't have one already. (https://cloud.digitalocean.com/registrations/new)
+
+2. **Web Application Code**: Have your web application code ready for production. This could be a Node.js, Python, Ruby on Rails, or any other type of application.
+
+Now that you have these two prerequisites, you're ready to launch your web app into the world! Follow these steps:
+
+### Step 1: Create a Droplet
+In digital ocean, a "Droplet" refers to a virtual private server (VPS) that you can deploy and manage on DigitalOcean's cloud infrastructure. It's essentially a scalable compute platform with add-on storage, where you can run applications, websites, and other services.
+
+1. Log in to your DigitalOcean account.
+2. On the left panel, click on "Droplets" under "Manage" and press "Create Droplet".
+
+Note that droplets do have a price per month to host on their platform, but DigitalOcean provides a free $200 budget for 60 days to newly created accounts.
+
+2. Choose a data center region. If you'd like the fastest speeds between your device and the web app, choose the closest region to you.
+3. Choose your operating system. Ubuntu is a popular choice for operating systems, as it is known to be fast.
+4. Select a plan based on your application's requirements. The CPUs and sizes that the platform offers differ in price.
+5. Optionally, you can also add an authentication method to your droplet to make it a private web app.
+6. Click "Create Droplet."
+
+<img src="https://github.com/learning-software-engineering/learning-software-engineering.github.io/assets/103388045/f45f232d-6eb4-4857-9e05-98e600d66791" height="500"/>
+
+### Step 2: Access Your Droplet
+Once the droplet is created, you will receive an email with login details.
+
+Use an SSH client to connect to your droplet. For example, run `ssh root@your_droplet_ip` in the terminal.
```

Comment:
Perhaps instead of "for example" we can just say we can accomplish this by running `ssh root...` in the terminal